.pipelines/e2e.yml

```diff
@@ -41,16 +41,29 @@ jobs:
 
       az account set -s $AZURE_SUBSCRIPTION_ID
 
+      sudo openvpn --config secrets/$VPN --daemon --writepid vpnpid
```

Comment:
Is `openvpn` guaranteed to be on the CI agents?

Also, since `kill_vpn` is being put into the hack/e2e script it may also make sense to include this VPN creation in there as well and move the script sourcing to earlier in the pipeline process.

Reply:
openvpn is guaranteed on CI agents as per https://github.com/Azure/ARO-RP/pull/2251
Added 'run_vpn' for ./hack/e2e/run-rp-and-e2e.sh 👍